skfusion/fusion/base/fusion_graph.py

```diff
@@ -421,6 +422,17 @@ def __contains__(self, obj_type):
     def __hash__(self):
         return hash(self.__str__())
 
+    def __eq__(self, other):
+
+        return (isinstance(other, self.__class__) and
+                self.name == other.name and
+                self.row_type == other.row_type and
+                self.col_type == other.col_type and
+                self.data.shape == other.data.shape)
```

Comment:
This is not a strict-enough condition for equality. But without it,
```python
G1 = FusionGraph()
rel = Relation(R12, ot1, ot2)
G1.add_relation(rel)
from copy import deepcopy
G2 = deepcopy(G1)
G2.remove_relation(rel)       # <--- this fails
```
```
File "/home/jk/Downloads/orange3/src/scikit-fusion/skfusion/fusion/base/fusion_graph.py", line 204, in remove_relation
  self.adjacency_matrix[relation.row_type][relation.col_type].remove(relation)
ValueError: list.remove(x): x not in list
```
In practice, relations and object types might be wrapped multiple times, being completely different Python objects of the same thing.

Problem?